packages/widgets/src/components/NotificationMenu/NotificationMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import { useTheme } from '@emotion/react';
+import {
+  Icon,
+  Popover,
+  PopoverClose,
+  PopoverContent,
+  PopoverDescription,
+  PopoverHeading,
+  PopoverTrigger,
+} from '@ssa-ui-kit/core';
+import { NotificationMenuProps } from './types';
+import {
+  ResetBtnStyles,
+  Close,
+  ButtonsWrapper,
+  List,
+  Loading,
+  PopoverWrapper,
+  PopoverContentWrapper,
+} from './styles';
+import { NotificationCard } from '@components/NotificationCard';
```

Comment:
Let's move this import before `./styles` and `./types` so that our imports follow this rule: 
1. 3rd party modules imports
2. Imports from the peer-projects, e.g. `@ssa-ui-kit/core`, `@ssa-ui-kit/utils`, etc.
3. The current project's  imports (`@components`, `@styles`, etc.)
4. The current component's imports (e.g., `./styles', `./types`, etc.)

Reply:
Please check other files follow this rule

Reply:
Done